install/kubernetes/helm/istio/values.yaml

```diff
@@ -238,7 +238,7 @@ sidecarInjectorWebhook:
 # galley configuration
 #
 galley:
-  enabled: false
+  enabled: true
```

Comment:
Please add _lots_ of comments. Users don't know or care about internal component names.